.github/workflows/summary.yaml

```diff
@@ -0,0 +1,84 @@
+name: Covasim Summary workflow
+on: [pull_request]
+
+jobs:
+  summary:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 8
+      matrix:
+        python-version: ['3.7']
+    name: Run and Summarize
+    steps:
+      - name: Set repo owner env variable
+        shell: python
+        run: print("::set-env name=GITHUB_OWNER::{}".format('${{github.repository}}'.split('/')[0]))
+      - name: Set PYTHONPATH
+        run: |
+          mkdir -p $HOME/.cache/site-packages
+          echo "::set-env name=PYTHONPATH::$HOME/.cache/site-packages"
+      - name: Checkout sources
+        uses: actions/checkout@v1
+      - uses: actions/setup-python@master
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+      - name: Cache Packages
+        id: cache-packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/ # This path is specific to Ubuntu
+          # Look to see if there is a cache hit for the corresponding requirements file
+          key: packages-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            packages-${{ runner.os }}-${{ matrix.python-version }}-
+      # Pip install first will make setup.py much faster and the cache will make pip install fast
+      - name: Install Covasim
+        run: |
+          python setup.py develop --install-dir ~/.cache/site-packages
+      # - name: Install synthpops
+      #   working-directory: ./synthpops
+      #   run: python setup.py develop
+      - name: Run Sim
+        env:
+          USE_POP_DATA: 0
+          DO_PLOT: 1
+          DO_SAVE: 1
+          DO_SUMMARY: 1
+        run: |
+          python examples/run_sim.py
+      - name: Set ENV Vars for Comment
+        id: comment-setup
+        run: python .github/workflows/support/diff.py
+      - name: Create Change comment
+        uses: jules2689/create-or-update-comment@acdf43d78d9100bbd1aa20960136760d1c463be4
+        if: steps.comment-setup.outputs.JSON_OUTPUT != ''
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          replace-new-line: true
+          comment-hash: 'summary_comment'
+          edit-mode: 'replace'
+          body: |
+            While unit tests and running simulations can help us validate nothing obvious has broken, the stochastic
+            nature of this model means that we can't explicitly test. This summary will help experts understand if we need to take a closer look.
+
+            <details>
```

Comment:
Should we hide the output in a by-default collapsed `details` block, or should we make it wide open to start?